### PR TITLE
Githubteacher kansascity

### LINKED ISSUE #19: GitHubTeacher planning for Kansas City file
I want to create a file with 
- place to eat
- [ ] things to do

### PR DESCRIPTION
**Pull request by githubteacher**
This pull request adds a file for things to do in Kansas City. 

This PR fixes #19 

@beardofedu What do you think I should add?

### CODE REVIEW

**Comment by beardofedu** [comment]
@githubteacher 
I was told I should check out the light and power district, should you add that to the list?
I like craft 🍺 and 🌮 where should I go?

img

File: githubteacher-kansascity.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Kansas City
+
+### Places to eat
```
Comment (githubteacher) [inline githubteacher-kansascity.md:3]: I'm hungry for BBQ!

Comment (githubteacher) [inline githubteacher-kansascity.md:3]: 🍕

File: githubteacher-kansascity.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+### Things to do
```
Comment (githubteacher) [inline githubteacher-kansascity.md:9]: 🍔

**Review by githubteacher — COMMENTED** [review]
💯

**Pull request merged**